.github/workflows/ci.yml

```
@@ -59,10 +59,374 @@ jobs:
+
+    # PVS-Studio Team: full analysis of the production package; fail CI when warnings remain.
+    # Secret: PVS_STUDIO_CREDENTIALS = "<license-name> <license-key>"
+    # Skipped on fork PRs (secrets unavailable). No SARIF upload.
+    # TEMPORARILY DISABLED: set back to the repository condition once the license is available.
+    pvs-studio:
+        name: PVS-Studio
+        runs-on: ubuntu-latest
+        timeout-minutes: 60
+        permissions:
+            contents: read
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)